cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1163,6 +1163,39 @@
   </property>
 
   <property>
+    <name>router.client.boss.threads</name>
+    <value>1</value>
+    <description>The number of boss threads in the router client</description>
+  </property>
+
+  <property>
+    <name>router.client.worker.threads</name>
```

Comment:
It would be a good idea to move all the configuration parameters that deal with router threads together.

@awholegunch Do you have a different order in which the parameters are specified in cdap-default.xml?